appium/lib/screen-object/accessors/element.rb

```diff
@@ -72,32 +71,30 @@ def element_attributes
       def get_position
         my_element = element.rect
```

Comment:
`my_element` is a weird name to choose 🤷‍♂

Reply:
changed